ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -59,6 +59,26 @@ Prediction.propTypes = {
   color: PropTypes.string,
   thresholds: PropTypes.arrayOf(PropTypes.number),
   href: PropTypes.string,
+  requiresCitation: PropTypes.bool,
+}
+
+const getPredictorFields = (variant, predictions, genePredictors) => {
+  const mappedFields = ORDERED_PREDICTOR_FIELDS.map(({
+    fieldTitle,
+    getHref,
+    ...predictorField
+  }) => ({
+    field: predictorField.field,
+    fieldTitle,
+    href: getHref && getHref(variant),
+    requiresCitation: predictorField.requiresCitation,
+    displaySortOrder: predictorField.displaySortOrder,
```

Comment:
I thought we had opted to remove `displaySortOrder`

Reply:
missed this, good catch